### ISSUE TITLE
Issue when install Desktop Video on Linux

### ISSUE THREAD

**Issue by a7med12h** [amiaopensource/vrecord#750]
I have Ubunto latest distribution. I have downloaded Desktop Video SW. and it has deb files . I tried to install packages and driver from files. It went ok but at the end the black magic SW could see Card. It means driver did not installed properly . Actually I trued by CMD and it was very very difficult. So I go by GUI deb .did any body installed Sesktop Video in Ubunto with no issue . Or please give best Linux distribution for installing this black magic driver

**Comment by privatezero** [comment]
Hi @a7med12h - I've always been successful with just using the standard `sudo apt install [input].deb` for installing Blackmagic Desktop Video on Ubuntu. For simplicity I usually use the package for `desktopvideo-gui`

**Comment by a7med12h** [comment]
> Hi @a7med12h - I've always been successful with just using the standard `sudo apt install [input].deb` for installing Blackmagic Desktop Video on Ubuntu. For simplicity I usually use the package for `desktopvideo-gui`


Yes , it must be easy . But I do not know why the software could not see the card. I checked the fan of card, It was runing. So it means the card has power. But why it did not recognized by software . I did installation by GUI . Also got same problem. Maybe my issue with card then !

**Comment by retokromer** [comment]
When you open Blackmagic Desktop Video Setup, then the home page displays your connected hardware and provides an overview of all video activity on your hardware’s input and output connections.

**Comment by a7med12h** [comment]
> When you open Blackmagic Desktop Video Setup, then the home page displays your connected hardware and provides an overview of all video activity on your hardware’s input and output connections.

Yes , you r right. That happened when I used Windows. But in Linux it did show me any connected devices

**Comment by privatezero** [comment]
Hopefully this is resolved via improvements to linux install process/newer versions of blackmagic software. Feel free to re-open if this is still an issue!
